Aggiunto capitolo CV per un developer (#112)
Il-Libro-Open-Source/book: docs/it/curriculum.md

_UN CV DI ESEMPIO_

Come anticipato in precedenza, di seguito trovi un esempio di un CV dal quale potrai prendere spunto, creato su Canva:

![CV Luke Skywalker](/book/assets/images/cv-luke.png)

2. La Foto
   Riguardo l’annosa questione “ma la foto la metto o no nel CV?” una risposta potrebbe essere "scegli tu". MA, se scegli di inserire una foto sul CV, ricorda:

- Preferibile una foto a mezzo busto, fatta in modo professionale e dove sei ritratto di fronte o leggermente di lato
- EVITA foto sgranate o molto vecchie
- EVITA foto che ti ritraggono al mare, ad una festa, con cocktail/cagnolini/gattini/cose in mano
- EVITA foto che sono evidentemente foto di gruppo che hai ritagliato (male)
- EVITA foto ammiccanti, con vestiti succinti o provocanti

Se non hai una tua foto professionale, va bene anche un avatar/illustrazione di te stesso, ma cerca di tenerti sempre nei limiti del professionale.

3. I Contatti
   Molto spesso i/le recruiter ricevono un sacco di CV senza, ad esempio, l’indirizzo mail e/o il numero di cellulare (ricorda sempre il prefisso internazionale, in Italia è +39).

>    Occhio quindi alla sezione contatti, perché è FONDAMENTALE.

Reviewer: ```suggestion
   Occhio quindi alla sezione "contatti", perché è FONDAMENTALE.
```

   Anche qui, alcune semplici regole:

> - La sezione contatti andrebbe inserita immediatamente sotto la foto, i contatti devono essere immediatamente visibili ed utilizzabili!

Reviewer: ```suggestion
- La sezione "contatti" andrebbe inserita immediatamente sotto la foto, i contatti devono essere immediatamente visibili ed utilizzabili!
```

4. L’About Me
   Questa sezione del CV è a dir poco FONDAMENTALE. Ti permetterà di riassumere in poche righe la tua esperienza professionale, le tecnologie/linguaggi che conosci o che stai approfondendo, i tuoi obiettivi lavorativi e personali e potresti anche fornire qualche spunto “extra” relativo magari a dei progetti personali che segui nel tempo libero, hobbies o altro che a tuo avviso potrebbe essere interessante.

L’about me è una delle prime cose che chi valuta CV legge e permette di avere una prima impressione di chi sei, cosa hai fatto e soprattutto cosa cerchi nel tuo futuro professionale. Nel CV di esempio di Luke potrai notare che l’about me ha sostanzialmente tutti gli elementi necessari: il job title attuale, le tecnologie/linguaggi che Luke conosce, segnala che gestisce un team di junior, scrive cosa cerca per il futuro. Inoltre, parla del suo interesse verso altre tematiche tecniche collegate al ruolo di Frontend ed infine fornisce informazioni su come impiega il suo tempo libero e quali sono suoi hobbies, che come vedremo nel paragrafo dedicato sono uno strumento interessante per un gancio di conversazione con chi dovrà valutarti.

5. Le Esperienze Professionali

>    Ovviamente le esperienze professionali sono il cuore di tutto il CV: in questa sezione l’obiettivo principale è di trovare il giusto compromesso fra l’essere sintetici ma allo stesso tempo esaustivi nel raccontare cosa hai fatto nel tuo percorso professionale e valorizzare i progetti più rilevanti, le tecnologie utilizzate, etc. Nell’esempio del CV di Luke noterai una certa standardizzazione del layout, questo perché il CV costruito in questo modo risulterà più leggibile, più snello ma allo stesso tempo completo di tutte le info utili, come ad esempio:

Reviewer: ```suggestion
   Ovviamente le esperienze professionali sono il cuore di tutto il CV: in questa sezione l’obiettivo principale è di trovare il giusto compromesso fra l’essere sintetici ma allo stesso tempo esaustivi nel raccontare cosa hai fatto nel tuo percorso professionale e valorizzare i progetti più rilevanti, le tecnologie utilizzate, ecc. Nell’esempio del CV di Luke noterai una certa standardizzazione del layout, questo perché il CV costruito in questo modo risulterà più leggibile, più snello ma allo stesso tempo completo di tutte le info utili, come ad esempio:
```

- Date di inizio e fine esperienza professionale, complete del nome dell’azienda

> - La dimostrazione di una crescita professionale coerente: nel cv di esempio il buon Luke inizia nel 2012 come FE Dev per poi crescere come Senior FE per poi arrivare all’ultima esperienza come Team Lead. Questa crescita professionale in termini di ruoli e responsabilità salta all’occhio di un recruiter/selezionatore esperto e fornisce un’overview

Reviewer: ```suggestion
- La dimostrazione di una crescita professionale coerente: nel CV di esempio il buon Luke inizia nel 2012 come FE Dev per poi crescere come Senior FE per poi arrivare all’ultima esperienza come Team Lead. Questa crescita professionale in termini di ruoli e responsabilità salta all’occhio di un recruiter/selezionatore esperto e fornisce un’overview
```

6. La Formazione
   Come potrai notare dal CV di esempio, per quanto riguarda il percorso di studi accademico è solo inserita la laurea, relativo voto e titolo della tesi (interessante perché potrebbe aiutare a tirar fuori un eventuale spunto di conversazione con il/la recruiter). In questo caso specifico, si tratta di un profilo senior quindi si è preferito inserire solo l’esperienza formativa relativa alla Laurea.

Se invece sei un profilo junior ha assolutamente senso inserire anche il percorso di diploma e relativa votazione (ovviamente se non hai conseguito la laurea, solo il diploma). Se stai frequentando l’Università, un PhD o la magistrale devi assolutamente inserirlo nel CV, per accedere ad uno stage curriculare ad esempio. In generale, ogni percorso formativo che si sta seguendo va segnalato nel CV per un doppio scopo: da un lato segnalare a chi valuta il CV che ti stai formando e seguendo un determinato percorso ed inoltre perché, sicuramente, avrai bisogno di un tot di ore di permessi studio per prepararti agli esami che poi dovrai sostenere, e questa è una informazione che chi ti sta valutando deve assolutamente avere!

> Stesso discorso se stai seguendo una Academy o un Bootcamp in ambito IT: è un percorso formativo (verticale sull’IT, ma pur sempre un percorso formativo) e va inserito, completo delle tecnologie che stai studiando, la data prevista di fine Academy ed i progetti pratici sui quali ti sei “sporcato” le mani.

Reviewer: ```suggestion
Stesso discorso se stai seguendo una Academy o un Bootcamp in ambito IT: è un percorso formativo (verticale sull’IT, ma pur sempre un percorso formativo) e va inserito, completo delle tecnologie che stai studiando, la data prevista di fine Academy ed i progetti pratici sui quali ti sei “sporcato/a” le mani.
```

7. Le Competenze Tecniche

>    Assieme alle esperienze professionali, il cuore di ogni CV, nel tuo caso questa sezione è decisamente importantissima. Ti permette chiaramente di elencare in maniera ordinata e sintetica tutte le competenze tecniche che hai acquisito (o che stai acquisendo, segnalandole magari con un “work in progress” o in maniera più grafica con una barra di avanzamento). ATTENZIONE: molte aziende usano gli ATS (spieghiamo dopo cos'è un ATS), che potrebbero non riconoscere segni grafici (tipo le stelline o le barre di avanzamento relative a quanta competenza si ha su quella tecnologia, consiglio quindi di non usare queste forme "grafiche").

Reviewer: ```suggestion
   Assieme alle esperienze professionali -l cuore di ogni CV- la sezione dedicata alle competenze tecniche è decisamente importantissima. Ti permette chiaramente di elencare in maniera ordinata e sintetica tutte le competenze tecniche che hai acquisito (o che stai acquisendo, segnalandole magari con un “work in progress” o in maniera più grafica con una barra di avanzamento). ATTENZIONE: molte aziende usano gli ATS (spieghiamo dopo cos'è), che potrebbero non riconoscere segni grafici (tipo le stelline o le barre di avanzamento relative a quanta competenza si ha su quella tecnologia, consiglio quindi di non usare queste forme "grafiche").
```

Sarebbe decisamente preferibile elencarle inserendo nelle prime posizioni le tecnologie/linguaggi sui quali sei più esperto/a o che usi giornalmente, in modo tale da non “confondere” chi sta leggendo il tuo CV (se ti candidi come sviluppatore frontend è decisamente sensato che come prime tecnologie/linguaggi in elenco ci siano linguaggi/framework inerenti al mondo frontend, per poi magari nelle ultime posizioni dell’elenco inserire linguaggi/framework di backend, database o altro).

> È inoltre estremamente utile per permettere agli ATS di attingere dalle keyword giuste relative alle tecnologie ricercate dall’azienda/selezionatore. L’ATS non è altro che un software che aiuta chi fa selezione in tutta una serie di attività ripetitive ed a basso valore aggiunto e funge anche come database di CV e candidature, oltre che al multiposting di un annuncio di lavoro su più portali contemporaneamente. È quindi importantissimo inserire nel CV tutte le parole chiave necessarie a far “risaltare” il CV fra i vari che l’azienda riceverà (sull’ATS ci sono una serie di falsi miti da sfatare assolutamente, in primis quello della “lettura automatica” del CV: i recruiter i CV li leggono tutti, ve lo assicuro. Semplicemente, l’ATS aiuta a filtrare i CV “più vicini” all’annuncio pubblicato).

Reviewer 2: ```suggestion
È inoltre estremamente utile per permettere agli ATS di attingere dalle keyword giuste relative alle tecnologie ricercate dall’azienda/selezionatore. L’ATS non è altro che un software che aiuta chi fa selezione in tutta una serie di attività ripetitive ed a basso valore aggiunto e funge anche come database di CV e candidature, oltre che al crossposting di un annuncio di lavoro su più portali contemporaneamente. È quindi importantissimo inserire nel CV tutte le parole chiave necessarie a far “risaltare” il CV fra i vari che l’azienda riceverà (sull’ATS ci sono una serie di falsi miti da sfatare assolutamente, in primis quello della “lettura automatica” del CV: i recruiter i CV li leggono tutti, ve lo assicuro. Semplicemente, l’ATS aiuta a filtrare i CV “più vicini” all’annuncio pubblicato).
```

> È inoltre estremamente utile per permettere agli ATS di attingere dalle keyword giuste relative alle tecnologie ricercate dall’azienda/selezionatore. L’ATS non è altro che un software che aiuta chi fa selezione in tutta una serie di attività ripetitive ed a basso valore aggiunto e funge anche come database di CV e candidature, oltre che al crossposting di un annuncio di lavoro su più portali contemporaneamente. È quindi importantissimo inserire nel CV tutte le parole chiave necessarie a far “risaltare” il CV fra i vari che l’azienda riceverà (sull’ATS ci sono una serie di falsi miti da sfatare assolutamente, in primis quello della “lettura automatica” del CV: i recruiter i CV li leggono tutti, ve lo assicuro. Semplicemente, l’ATS aiuta a filtrare i CV “più vicini” all’annuncio pubblicato).

Reviewer: ```suggestion
È inoltre estremamente utile per permettere agli ATS di attingere dalle keyword giuste relative alle tecnologie ricercate dall’azienda/selezionatore. L’ATS non è altro che un software che aiuta chi fa selezione in tutta una serie di attività ripetitive ed a basso valore aggiunto e funge anche come database di CV e candidature, oltre che al crossposting di un annuncio di lavoro su più portali contemporaneamente. È quindi importantissimo inserire nel CV tutte le parole chiave necessarie a far “risaltare” il CV fra i vari che l’azienda riceverà.

Nota: sull’ATS ci sono una serie di falsi miti da sfatare assolutamente, in primis quello della “lettura automatica” del CV: i recruiter i CV li leggono tutti, ve lo assicuro. Semplicemente, l’ATS aiuta a filtrare i CV “più vicini” all’annuncio pubblicato.
```

8. Le Soft Skills
   Riguardo le soft skills ci sono varie scuole di pensiero: c’è chi pensa che per un lavoro “tecnico” come quello del dev siano inutili, c’è chi pensa che invece siano fondamentali. Personalmente, trovo che un/una qualsiasi professionista (quindi non solo un/una dev) debba costruire la propria professionalità al 50% su hard skills ed al 50% su soft skills: non si può pensare di essere estremamente forti tecnicamente ma mancare totalmente di capacità comunicative, teamworking, empatia, propositività e molto altro. Contrariamente a quello che si pensa, il lavoro del/della dev non è un lavoro robotico, ma ha bisogno di una grandissima componente “umana” che risiede proprio nelle soft skills, che vanno nutrite e costruite, esattamente come le hard skills.

> E' quindi importantissimo inserirle nel CV ma con un piccolo promemoria: BASTA con le soft skills trite e ritrite! Cerca di scrivere un elenco di soft skill “reali” o comunque calate davvero sulla tua personalità e modo di essere: se sei ad esempio una persona molto meticolosa e che ha bisogno di ordine e routine, probabilmente non sarai una persona che lavora bene sotto stress o in velocità (e non c’è nulla di male a non essere quel tipo di persona), quindi scrivere “lavoro bene sotto pressione” è falso (oltre che una delle soft skill trite e ritrite di cui sopra).

Reviewer: ```suggestion
È quindi importantissimo inserirle nel CV ma con un piccolo promemoria: BASTA con le soft skills trite e ritrite! Cerca di scrivere un elenco di soft skill “reali” o comunque calate davvero sulla tua personalità e modo di essere: se sei ad esempio una persona molto meticolosa e che ha bisogno di ordine e routine, probabilmente non sarai una persona che lavora bene sotto stress o in velocità (e non c’è nulla di male a non essere quel tipo di persona), quindi scrivere “lavoro bene sotto pressione” è falso (oltre che una delle soft skill trite e ritrite di cui sopra).
```

Reviewer: Alt+0200, maledettooo

Nel CV di esempio, chiaramente, le soft skills inserite sono volutamente goliardiche (come quasi tutto il resto del CV d’altronde). Ma riguardo questa sezione mi piacerebbe che tu facessi un piccolo “esercizio mentale” e trovassi le soft skill che, a tuo avviso, ti caratterizzano davvero. Possono essere un ottimo spunto di conversazione con chi ti valuterà!

9. Corsi e Certificazioni
   Molte aziende, come requisito obbligatorio per alcune posizioni, richiedono delle certificazioni specifiche. Di conseguenza, questa sezione è importante proprio per segnalare di aver completato determinati corsi o avere in mano delle certificazioni riconosciute, spesso anche a livello globale. È inoltre una sezione utile per dimostrare il tuo impegno personale e la volontà di accrescere le tue competenze tecniche.

Capita spesso, ad esempio, di trovare nei CV corsi e certificazioni sullo Scrum/Agile/PM che possono essere usate come spunto di conversazione: spesso denotano una volontà di voler crescere professionalmente e quindi un segnale del tipo: “hey vedi, ho una certificazione da PM, è quello che voglio fare ‘da grande’”!

10. Lingue e Hobby

>     Il 90% dei CV che normalmente i/le recruiter ricevono NON ha la sezione hobby, ed è un peccato! È invece una delle sezioni più interessanti (ed in alcuni casi, divertente) di tutto il CV. Ti permette di farti conoscere come persona, e non solo come tecnico. Fornisce inoltre un sacco di spunti di conversazione utili a carpire ulteriori punti di vista, modi di vedere le cose, obiettivi raggiunti e molto altro.

Reviewer: ```suggestion
    Il 90% dei CV che normalmente i/le recruiter ricevono NON ha la sezione hobby, ed è un peccato! È invece una delle sezioni più interessanti (ed in alcuni casi, divertente) di tutto il CV. Ti permette di farti conoscere come persona, e non solo come professionista. Fornisce inoltre un sacco di spunti di conversazione utili a carpire ulteriori punti di vista, modi di vedere le cose, obiettivi raggiunti e molto altro.
```

> Ad esempio, una persona che pratica sport estremi avrà probabilmente una sana propensione a superare i propri limiti, la curiosità di provare cose nuove e sicuramente coraggio. Una persona che pratica sport di squadra invece sarà sicuramente abituata a lavorare in team, a collaborare e tenderà a non essere un “lupo solitario”. Gli hobby raccontano tantissimo di te, inseriscili nel CV! Riguardo le lingue poco da dire: siamo nel 2024, è FONDAMENTALE quanto meno conoscere un’altra lingua (preferibilmente l’inglese) oltre all’italiano. Anche in questo caso, eventuali certificazioni sulle lingue, se le hai, inseriscile nel CV in questa sezione.

Reviewer 2: Mi permetto di aggiungere una riflessione

```suggestion
Ad esempio, una persona che pratica sport estremi avrà probabilmente una sana propensione a superare i propri limiti, la curiosità di provare cose nuove e sicuramente coraggio. Una persona che pratica sport di squadra invece sarà sicuramente abituata a lavorare in team, a collaborare e tenderà a non essere un “lupo solitario”. Gli hobby raccontano tantissimo di te, inseriscili nel CV! Riguardo le lingue poco da dire: siamo nel 2024, è FONDAMENTALE quanto meno conoscere un’altra lingua (preferibilmente l’inglese) oltre all’italiano. Anche in questo caso, eventuali certificazioni sulle lingue, se le hai, inseriscile nel CV in questa sezione. Evitiamo tuttavia di inserire lingue che a malapena sappiamo parlare e comprendere, che magari abbiamo studiato nei primi anni di scuola.
```

Pur non essendo obbligatorio inserire l’autorizzazione al trattamento dei dati nel CV, è utile farlo perché si permette immediatamente al datore di lavoro di usare i propri dati personali per essere contattati per un colloquio oppure di conservarli per future selezioni.
Se il datore di lavoro riceve un CV privo di autorizzazione, deve richiedere al candidato l’autorizzazione fornendo tutti i dettagli sulle modalità di utilizzo dei dati. Solo dopo aver ricevuto questo consenso potrà utilizzarli, di conseguenza come puoi ben immaginare questo porta a lungaggini nell’iter di selezione e introduce un ulteriore strato di burocrazia che potrebbe essere evitato.

12. Considerazioni Finali

Ricorda che il CV è una vetrina di te stesso/a: delle tue esperienze professionali prima di tutto, ma anche del tuo percorso formativo, delle tue passioni, delle tue soft skills e della tua volontà di trovare il “posto giusto” per te.

> Il CV non va visto quindi come una sterile lista della spesa di tecnologie/linguaggi/certificazioni ma in qualche modo deve riuscire a raccontare te stesso, in maniera certamente sintetica ma coerente. È il primo biglietto da visita di te stesso che arriva nelle mani delle persone che, probabilmente, ti assumeranno e con le quali lavorerai. Quindi, investi qualche ora nella costruzione del tuo CV e non te ne pentirai!

Reviewer 3: ```suggestion
Il CV non va visto quindi come una sterile lista della spesa di tecnologie/linguaggi/certificazioni ma in qualche modo deve riuscire a raccontare te stesso/a, in maniera certamente sintetica ma coerente. È il primo biglietto da visita di te stesso/a che arriva nelle mani delle persone che, probabilmente, ti assumeranno e con le quali lavorerai. Quindi, investi qualche ora nella costruzione del tuo CV e non te ne pentirai!
```

# Il Curriculum

> Se stai leggendo questo capitolo probabilmente ti stai guardando attorno, alla ricerca di un nuovo lavoro, una nuova collaborazione, una nuova RAL (si spera più alta dell'attuale).

Author: fatto

- Personalizza! L'europass è un formato ormai datato e senza alcuna possibilità di personalizzazione grafica. L'esempio che troverai in basso è stato creato con [Canva](https://www.canva.com/), contenente centinaia di template validi che puoi scegliere.
  Potresti usare anche [LaTeX](https://www.latextemplates.com/) oppure [OverLeaf](https://www.overleaf.com/))

Aggiungo inoltre alcune considerazioni più "personali" sulle quali dovresti riflettere, prendendo spunto da un post di Roberta Zantedeschi, che trovi [QUI]
(https://www.linkedin.com/posts/robertazantedeschi_cv-curriculum-parolepersonelavoro-activity-7034528956879974400-kGoE?utm_source=share&utm_medium=member_desktop)

Quali sono i principali ostacoli che potresti incontrare quando scrivi il tuo CV?

- in primis poca connessione e conoscenza di te
- scarsa stima del tuo valore e delle tue risorse
- idee poco chiare rispetto a ciò che stai cercando, a chi stai parlando, alla direzione che vuoi perseguire

> - urgenza (scriverlo quando sei nel bisogno e non nella presenza a te e ai tuoi obiettivi)

Author: è roba filosofica della roberta :D

_UN CV DI ESEMPIO_

Come anticipato in precedenza, di seguito trovi un esempio di un CV dal quale potrai prendere spunto, creato su Canva:

![CV-Luke-Skywalker-FE-Developer_v3.png](./CV-Luke-Skywalker-FE-Developer_v3.png)

2. La Foto
   Riguardo l’annosa questione “ma la foto la metto o no nel CV?” una risposta potrebbe essere "scegli tu". MA, se scegli di inserire una foto sul CV, ricorda:

- Preferibile una foto a mezzo busto, fatta in modo professionale e dove sei ritratto di fronte o leggermente di lato
- EVITA foto sgranate o molto vecchie
- EVITA foto che ti ritraggono al mare, ad una festa, con cocktail/cagnolini/gattini/cose in mano
- EVITA foto che sono evidentemente foto di gruppo che hai ritagliato (male)
- EVITA foto ammiccanti, con vestiti succinti o provocanti

Se non hai una tua foto professionale, va bene anche un avatar/illustrazione di te stesso, ma cerca di tenerti sempre nei limiti del professionale.

3. I Contatti

>    Molto spesso i/le recruiter ricevono un sacco di CV senza, ad esempio, l’indirizzo mail e/o il numero di cellulare.

Reviewer 4: Propongo:

```suggestion
   Molto spesso i/le recruiter ricevono un sacco di CV senza, ad esempio, l’indirizzo mail e/o il numero di cellulare (ricorda sempre il prefisso internazionale, in Italia è +39).
```

   Occhio quindi alla sezione contatti, perché è FONDAMENTALE.
   Anche qui, alcune semplici regole:

- La sezione contatti andrebbe inserita immediatamente sotto la foto, i contatti devono essere immediatamente visibili ed utilizzabili!
- Ricorda di inserire tutti i contatti utili, come nel CV di esempio
- Scegli un indirizzo mail professionale
- Inserisci i link del tuo profilo LinkedIn, GitHub (e se hai portfolio online anche di quello). Rendi questi link cliccabili per agevolare chi legge il tuo CV ad una rapida apertura del link senza dover fare copia-incolla ogni volta

4. L’About Me
   Questa sezione del CV è a dir poco FONDAMENTALE. Ti permetterà di riassumere in poche righe la tua esperienza professionale, le tecnologie/linguaggi che conosci o che stai approfondendo, i tuoi obiettivi lavorativi e personali e potresti anche fornire qualche spunto “extra” relativo magari a dei progetti personali che segui nel tempo libero, hobbies o altro che a tuo avviso potrebbe essere interessante.

> L’about me è una delle prime cose che chi valuta CV legge e permette di avere una prima overview di chi sei, cosa hai fatto e soprattutto cosa cerchi nel tuo futuro professionale. Nel CV di esempio di Luke potrai notare che l’about me ha sostanzialmente tutti gli elementi necessari: il job title attuale, le tecnologie/linguaggi che Luke conosce, segnala che gestisce un team di junior, scrive cosa cerca per il futuro. Inoltre, parla del suo interesse verso altre tematiche tecniche collegate al ruolo di FE ed infine fornisce una overview del suo tempo libero e dei suoi hobbies, che come vedremo nel paragrafo dedicato sono uno strumento interessante per un gancio di conversazione con chi dovrà valutarti.

Reviewer 4: ```suggestion
L’about me è una delle prime cose che chi valuta CV legge e permette di avere una prima impressione di chi sei, cosa hai fatto e soprattutto cosa cerchi nel tuo futuro professionale. Nel CV di esempio di Luke potrai notare che l’about me ha sostanzialmente tutti gli elementi necessari: il job title attuale, le tecnologie/linguaggi che Luke conosce, segnala che gestisce un team di junior, scrive cosa cerca per il futuro. Inoltre, parla del suo interesse verso altre tematiche tecniche collegate al ruolo di Frontend ed infine fornisce una informazioni su come impiega il suo tempo libero e quali sono suoi hobbies, che come vedremo nel paragrafo dedicato sono uno strumento interessante per un gancio di conversazione con chi dovrà valutarti.
```

5. Le Esperienze Professionali
   Ovviamente le esperienze professionali sono il cuore di tutto il CV: in questa sezione l’obiettivo principale è di trovare il giusto compromesso fra l’essere sintetici ma allo stesso tempo esaustivi nel raccontare cosa hai fatto nel tuo percorso professionale e valorizzare i progetti più rilevanti, le tecnologie utilizzate, etc. Nell’esempio del CV di Luke noterai una certa standardizzazione del layout, questo perché il CV costruito in questo modo risulterà più leggibile, più snello ma allo stesso tempo completo di tutte le info utili, come ad esempio:

- Date di inizio e fine esperienza professionale, complete del nome dell’azienda
- La dimostrazione di una crescita professionale coerente: nel cv di esempio il buon Luke inizia nel 2012 come FE Dev per poi crescere come Senior FE per poi arrivare all’ultima esperienza come Team Lead. Questa crescita professionale in termini di ruoli e responsabilità salta all’occhio di un recruiter/selezionatore esperto e fornisce un’overview

> - Bullet point che descrivono sinteticamente ma in maniera efficace le mansioni principali, i progetti seguiti, le tecnologie utilizzate ed altre informazioni a tuo avviso rilevanti per la specifica posizione che stai ricercando

Reviewer 4: ```suggestion
- Elenchi puntati che descrivono sinteticamente ma in maniera efficace le mansioni principali, i progetti seguiti, le tecnologie utilizzate ed altre informazioni a tuo avviso rilevanti per la specifica posizione che stai ricercando
```

6. La Formazione
   Come potrai notare dal CV di esempio, per quanto riguarda il percorso di studi accademico è solo inserita la laurea, relativo voto e titolo della tesi (interessante perché potrebbe aiutare a tirar fuori un eventuale spunto di conversazione con il/la recruiter). In questo caso specifico, si tratta di un profilo senior quindi si è preferito inserire solo l’esperienza formativa relativa alla Laurea.

Se invece sei un profilo junior ha assolutamente senso inserire anche il percorso di diploma e relativa votazione (ovviamente se non sei laureato, solo il diploma). Se stai frequentando l’Università, un PhD o la magistrale devi assolutamente inserirlo nel CV, per accedere ad uno stage curriculare ad esempio. In generale, ogni percorso formativo che si sta seguendo va segnalato nel CV per un doppio scopo: da un lato segnalare a chi valuta il CV che ti stai formando e seguendo un determinato percorso ed inoltre perché, sicuramente, avrai bisogno di un tot di ore di permessi studio per prepararti agli esami che poi dovrai sostenere, e questa è una informazione che chi ti sta valutando deve assolutamente avere!

> Idem se stai seguendo una Academy/Bootcamp in ambito IT: è un percorso formativo (verticale sull’IT, ma pur sempre un percorso formativo) e va inserito, completo delle tecnologie che stai studiando, la data prevista di fine Academy ed il progetto/i progetti pratici sui quali ti sei “sporcato” le mani.

Reviewer 4: ```suggestion
Stessa discorso se stai seguendo una Academy o un Bootcamp in ambito IT: è un percorso formativo (verticale sull’IT, ma pur sempre un percorso formativo) e va inserito, completo delle tecnologie che stai studiando, la data prevista di fine Academy ed i progetti pratici sui quali ti sei “sporcato” le mani.
```

7. Le Competenze Tecniche

>    Assieme alle esperienze professionali, il cuore di ogni CV, nel tuo caso questa sezione è decisamente importantissima. Ti permette chiaramente di elencare in maniera ordinata e sintetica tutte le competenze tecniche che hai acquisito (o che stai acquisendo, segnalandole magari con un “work in progress” o in maniera più grafica con una barra di avanzamento). ATTENZIONE: molte aziende usano gli ATS, che potrebbero non riconoscere segni grafici (tipo le stelline o le barre di avanzamento relative a quanta competenza si ha su quella tecnologia).

Reviewer 4: Sinceramente non sono un amante delle barre di avanzamento perchè non forniscono, secondo me, un feedback adeguato.
Molto meglio utilizzare delle scale del tipo: beginner, intermediate, advanced, expert. (Mai mettere expert, il recruiter potrebbe vederla come una sfida, farlo solo se è un tool/libreria che hai scritto tu).

Mi è capitato molto spesso di vedere CV con su scritto:
* HTML 80%
* CSS 60% 
* ...

Ma non ho mai capito il senso delle percentuali sulle competenze tecniche.

Author: fatto

> Sarebbe decisamente preferibile elencarle inserendo nelle prime posizioni le tecnologie/linguaggi sui quali sei più esperto/a o che usi giornalmente, in modo tale da non “confondere” chi sta leggendo il tuo CV (se ti candidi come FE è decisamente sensato che come prime tecnologie/linguaggi in elenco ci siano linguaggi/FW di FE, per poi magari nelle ultime posizioni dell’elenco inserire linguaggi/FW di BE, database o altro).

Reviewer 4: ```suggestion
Sarebbe decisamente preferibile elencarle inserendo nelle prime posizioni le tecnologie/linguaggi sui quali sei più esperto/a o che usi giornalmente, in modo tale da non “confondere” chi sta leggendo il tuo CV (se ti candidi come sviluppatore frontend è decisamente sensato che come prime tecnologie/linguaggi in elenco ci siano linguaggi/framework inerenti al mondo frontend, per poi magari nelle ultime posizioni dell’elenco inserire linguaggi/framework di backend, database o altro).
```

> È inoltre estremamente utile per permettere agli ATS di attingere dalle keyword giuste relative alle tecnologie ricercate dall’azienda/selezionatore. L’ATS non è altro che un software che aiuta chi fa selezione in tutta una serie di attività ripetitive ed a basso valore aggiunto e funge anche come database di CV e candidature, oltre che al multiposting di un annuncio di lavoro su più portali contemporaneamente. È quindi importantissimo inserire nel CV tutte le parole chiave necessarie a far “risaltare” il CV fra i vari che l’azienda riceverà (sull’ATS ci sono una serie di falsi miti da sfatare assolutamente, in primis quello della “lettura automatica” del CV: i recruiter i CV li leggono tutti, ve lo assicuro. Semplicemente, l’ATS aiuta a filtrare i CV “più vicini” all’annuncio pubblicato).

Reviewer 4: Valuta se fornire una definizione più attenta di ATS quando viene utilizzato sopra per la prima volta che compare nel testo così da non doverlo ripetere qui.

Author: fatto

8. Le Soft Skills
   Riguardo le soft skills ci sono varie scuole di pensiero: c’è chi pensa che per un lavoro “tecnico” come quello del dev siano inutili, c’è chi pensa che invece siano fondamentali. Personalmente, trovo che un/una qualsiasi professionista (quindi non solo un/una dev) debba costruire la propria professionalità al 50% su hard skills ed al 50% su soft skills: non si può pensare di essere estremamente forti tecnicamente ma mancare totalmente di capacità comunicative, teamworking, empatia, propositività e molto altro. Contrariamente a quello che si pensa, il lavoro del/della dev non è un lavoro robotico, ma ha bisogno di una grandissima componente “umana” che risiede proprio nelle soft skills, che vanno nutrite e costruite, esattamente come le hard skills.

> E' quindi importantissimo inserirle nel CV ma con un piccolo reminder: BASTA con le soft skills trite e ritrite! Cerca di scrivere un elenco di soft skill “reali” o comunque calate davvero sulla tua personalità e modo di essere: se sei ad esempio una persona molto meticolosa e che ha bisogno di ordine e routine, probabilmente non sarai una persona che lavora bene sotto stress o in velocità (e non c’è nulla di male a non essere quel tipo di persona), quindi scrivere “lavoro bene sotto pressione” è falso (oltre che una delle soft skill trite e ritrite di cui sopra).

Reviewer 4: ```suggestion
E' quindi importantissimo inserirle nel CV ma con un piccolo promemoria: BASTA con le soft skills trite e ritrite! Cerca di scrivere un elenco di soft skill “reali” o comunque calate davvero sulla tua personalità e modo di essere: se sei ad esempio una persona molto meticolosa e che ha bisogno di ordine e routine, probabilmente non sarai una persona che lavora bene sotto stress o in velocità (e non c’è nulla di male a non essere quel tipo di persona), quindi scrivere “lavoro bene sotto pressione” è falso (oltre che una delle soft skill trite e ritrite di cui sopra).
```

Nel CV di esempio, chiaramente, le soft skills inserite sono volutamente goliardiche (come quasi tutto il resto del CV d’altronde). Ma riguardo questa sezione mi piacerebbe che tu facessi un piccolo “esercizio mentale” e trovassi le soft skill che, a tuo avviso, ti caratterizzano davvero. Possono essere un ottimo spunto di conversazione con chi ti valuterà!

9. Corsi e Certificazioni
   Molte aziende, come requisito obbligatorio per alcune posizioni, richiedono delle certificazioni specifiche. Di conseguenza, questa sezione è importante proprio per segnalare di aver completato determinati corsi o avere in mano delle certificazioni riconosciute, spesso anche a livello globale. È inoltre una sezione utile per dimostrare il tuo impegno personale e la volontà di accrescere le tue competenze tecniche.

> Capita spesso, ad esempio, di trovare nei CV corsi e certificazioni sullo Scum/Agile/PM che possono essere usate come spunto di conversazione: spesso denotano una volontà di voler crescere professionalmente e quindi un segnale del tipo: “hey vedi, ho una certificazione da PM, è quello che voglio fare ‘da grande’”!

Reviewer 4: ```suggestion
Capita spesso, ad esempio, di trovare nei CV corsi e certificazioni sullo Scrum/Agile/PM che possono essere usate come spunto di conversazione: spesso denotano una volontà di voler crescere professionalmente e quindi un segnale del tipo: “hey vedi, ho una certificazione da PM, è quello che voglio fare ‘da grande’”!
```

10. Lingue e Hobby

>     Il 90% dei CV che normalmente i/le recruiter ricevono NON ha la sezione hobby, ed è un peccato! E' invece una delle sezioni più interessanti (ed in alcuni casi, divertente) di tutto il CV. Ti permette di farti conoscere come come persona, e non solo come tecnico. Fornisce inoltre un sacco di spunti di conversazione utili a carpire ulteriori punti di vista, modi di vedere le cose, obiettivi raggiunti e molto altro.

Reviewer 4: ```suggestion
    Il 90% dei CV che normalmente i/le recruiter ricevono NON ha la sezione hobby, ed è un peccato! È invece una delle sezioni più interessanti (ed in alcuni casi, divertente) di tutto il CV. Ti permette di farti conoscere come persona, e non solo come tecnico. Fornisce inoltre un sacco di spunti di conversazione utili a carpire ulteriori punti di vista, modi di vedere le cose, obiettivi raggiunti e molto altro.
```

> Ad esempio, una persona che pratica sport estremi avrà probabilmente una sana propensione a superare i propri limiti, la curiosità di provare cose nuove e sicuramente coraggio. Una persona che pratica sport di squadra invece sarà sicuramente abituata a lavorare in team, a collaborare e tenderà a non essere un “lupo solitario”. Gli hobby raccontano tantissimo di te, inseriscili nel CV! Riguardo le lingue poco da dire: siamo nel 2023, è FONDAMENTALE quanto meno conoscere un’altra lingua (preferibilmente l’inglese) oltre all’italiano. Anche in questo caso, eventuali certificazioni sulle lingue, se le hai, inseriscile nel CV in questa sezione.

Reviewer 4: ```suggestion
Ad esempio, una persona che pratica sport estremi avrà probabilmente una sana propensione a superare i propri limiti, la curiosità di provare cose nuove e sicuramente coraggio. Una persona che pratica sport di squadra invece sarà sicuramente abituata a lavorare in team, a collaborare e tenderà a non essere un “lupo solitario”. Gli hobby raccontano tantissimo di te, inseriscili nel CV! Riguardo le lingue poco da dire: siamo nel 2024, è FONDAMENTALE quanto meno conoscere un’altra lingua (preferibilmente l’inglese) oltre all’italiano. Anche in questo caso, eventuali certificazioni sulle lingue, se le hai, inseriscile nel CV in questa sezione.
```

11. L’Autorizzazione al Trattamento Dati
    Il Decreto Legislativo 10 agosto 2018, n. 101 ha stabilito che il consenso al trattamento dei dati personali presenti nel CV non è più dovuto. Questo significa che in base alla nuova normativa sulla privacy, non è obbligatorio includere il consenso al trattamento dei dati nel Curriculum Vitae. Ciascun candidato può scegliere liberamente di inserirlo o meno. Detto questo, consiglio comunque di inserire la frase che puoi tranquillamente copiare dal CV di esempio per diversi motivi:

> Inserire l’Autorizzazione al Trattamento dei Dati Personali nel proprio Curriculum Vitae significa autorizzare chi riceve il CV (il datore di lavoro, i responsabili Risorse Umane e chi si occupa della selezione) al trattamento dei dati coperti e tutelati dalla legge sulla Privacy italiana e dal GDPR – Regolamento Generale sulla Protezione Dati, in vigore in tutti i paesi dell’Unione Europea dal 25 maggio 2018

Reviewer 4: ```suggestion
Inserire l’Autorizzazione al Trattamento dei Dati Personali nel proprio Curriculum Vitae significa autorizzare chi riceve il CV (il datore di lavoro, i responsabili Risorse Umane e chi si occupa della selezione) al trattamento dei dati coperti e tutelati dalla legge sulla Privacy italiana e dal GDPR – Regolamento Generale sulla Protezione Dati, in vigore in tutti i paesi dell’Unione Europea dal 25 maggio 2018.
```

> Pur non essendo obbligatorio inserire l’autorizzazione al trattamento dei dati nel CV, è utile farlo perché si permette immediatamente al datore di lavoro di usare i propri dati personali per essere contattati per un colloquio oppure di conservarli per future selezioni

Reviewer 4: ```suggestion
Pur non essendo obbligatorio inserire l’autorizzazione al trattamento dei dati nel CV, è utile farlo perché si permette immediatamente al datore di lavoro di usare i propri dati personali per essere contattati per un colloquio oppure di conservarli per future selezioni.
```

> Se il datore di lavoro riceve un CV privo di autorizzazione, deve richiedere al candidato l’autorizzazione fornendo tutti i dettagli sulle modalità di utilizzo dei dati. Solo dopo aver ricevuto questo consenso potrà utilizzarli, di conseguenza come puoi ben immaginare questo porta a lungaggini nell’iter di selezione ed un ulteriore “layer” di burocrazia che si può tranquillamente evitare!

Reviewer 4: ```suggestion
Se il datore di lavoro riceve un CV privo di autorizzazione, deve richiedere al candidato l’autorizzazione fornendo tutti i dettagli sulle modalità di utilizzo dei dati. Solo dopo aver ricevuto questo consenso potrà utilizzarli, di conseguenza come puoi ben immaginare questo porta a lungaggini nell’iter di selezione e introduce un ulteriore strato di burocrazia che potrebbe essere evitato.
```
